server-spi/src/main/java/org/keycloak/models/RealmProvider.java

```diff
@@ -200,6 +230,6 @@ default ClientModel addClient(RealmModel realm, String clientId) {
     /**
      * @deprecated Use the corresponding method from {@link ClientProvider}. */
     @Override
-    public List<ClientModel> getAlwaysDisplayInConsoleClients(RealmModel realm);
+    public Stream<ClientModel> getAlwaysDisplayInConsoleClientsStream(RealmModel realm);
```

Comment:
In case I revert this I have to implement the method in `JpaRealmProvider` and `RealmCacheSession` or the method has to be removed from `RealmProvider`.

Reply:
I proceeded with removing the method from `RealmProvider`.